common_spree_dependencies.rb

```diff
@@ -14,28 +14,38 @@
 end
 
 platforms :ruby do
-  gem 'mysql2'
+  gem 'mysql2', '~> 0.4.10'
   gem 'pg', '~> 0.18'
 end
 
 group :test do
-  gem 'capybara', '~> 2.16'
+  gem 'bootstrap-sass', '>= 3.3.5.1', '< 3.4'
+  gem 'capybara', '~> 2.4'
   gem 'capybara-screenshot', '~> 1.0'
+  gem 'canonical-rails', '~> 0.2.0'
   gem 'database_cleaner', '~> 1.3'
   gem 'email_spec'
   gem 'factory_bot_rails', '~> 4.8'
+  gem 'jquery-rails', '~> 4.3'
+  gem 'jquery-ui-rails', '~> 6.0.1'
   gem 'launchy'
+  gem 'poltergeist', '~> 1.10'
```

Comment:
@MatthewKennedy we're not using poltergeist as of https://github.com/spree/spree/pull/8429